packages/docs/components/tab.md

```diff
@@ -80,7 +80,7 @@ see what could be grouped together. Those themes should become your Tabs.
 
 <Description>
 
-**Tabs are not navigation**. Meaning they don’t take you from place to place. Rather, they are meant for context switching related to the page.
+_Tabs are not navigation_. Meaning they don’t take you from place to place. Rather, they are meant for context switching related to the page.
```

Comment:
I'd prefer we left out most whole-sentence formatting. It's not present in most of the docs, and I'm not sure this is much stronger than our other guidelines.

Reply:
```suggestion
Tabs are not navigation. Meaning they don’t take you from place to place. Rather, they are meant for context switching related to the page.
```